### DISCUSSION TITLE
How can i see thumbnails of all videos for a specific channel?

### DISCUSSION THREAD

**Discussion by LinArcX** [pystardust/ytfzf#686] [Q&A] [answered]
I want to see all thumbnails of this channel: @TsodingDaily

So what i tried so far:
`yt.sh -c youtube-channel @TsodingDaily --upload-date=year`

It just shows me last 30 videos.

Or even i tried:
`yt.sh -c youtube-channel @TsodingDaily --pages=110`

But still i see last 30 video thumbnails in my fzf.

Any  idea?

**Comment by Euro20179** [comment, marked as answer]
You'll need to use the channel id for this, to get the channel id do `ytfzf --channel-link=@tsodingdaily`

When you get that, you can do `ytfzf --pages=3 -c invidious-playlist https://www.youtube.com/playlist?list=<channel-id>`

**Comment by LinArcX** [comment]
Thanks. i wrote this little snippet for everyone who is interested:

```
      channels=("@TsodingDaily" "@MollyRocket" "@TEDx" "@oxforduniversity")
      selected_channel=$(printf '%s\n' "${channels[@]}" | fzf --header="channels:")
      channel_raw=$(ytfzf.sh --channel-link=$selected_channel)
      channel_id=$(echo "$channel_raw" | sed 's#.*/##')
      ~/bin/scripts/ytfzf.sh --pages=3 -c invidious-playlist https://www.youtube.com/playlist?list=$channel_id
 
```
